packages/zmq-android.4.0-8/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "anders@fugmann.net"
+homepage: "https://github.com/issuu/ocaml-zmq"
+authors: [ "Anders Fugman" "Pedro Borges" "Peter Zotov" ]
+license: "MIT"
+dev-repo: "https://github.com/issuu/ocaml-zmq.git"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+build: [
+  ["env" "OCAMLFIND_TOOLCHAIN=android" "ocaml" "setup.ml" "-configure" "--prefix" "%{prefix}%/android-sysroot"]
+  ["env" "OCAMLFIND_TOOLCHAIN=android" "ocaml" "setup.ml" "-build"]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=android" "ocaml" "setup.ml" "-install"]
+]
+remove: [
+  ["ocamlfind" "-toolchain" "android" "remove" "ZMQ"]
+]
+depends: [
+  "ocaml-android"
+  "conf-zmq-android"
```

Comment:
Is MXE means http://mxe.cc/ ?
I don't use android-ndk -- I'm sit on yocto/bitbake with plain gcc cross-compiler suite, so I can't use it.